mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/6 main menu redesign

Closes #6 
Closes #21 

To test:
- Run Main Menu scene
- Set the CyberCIEGE install folder in preferences dialog that automatically pops up
- Make sure progress bar (slider to right of big title image) matches value in Java CyberCIEGE Launcher
- Make sure correct campaigns/scenarios are locked (see note below)
- Select Starting Scenarios campaign, Introduction Scenario scenario, New Game saved game, and click Play
- Make sure game plays as before
- Make sure headless CyberCIEGE window closes when Unity app is exited
- Run Main Menu scene
- Make sure preferences dialog does NOT pop up and same campaign, scenario, and saved game are selected

Notes:
- Campaign/scenarios are locked solely based on their prerequisite data in their associated campaign file and if they don't have any, they are unlocked.  The Java app seems to be using slightly different rules since Navy campaign is locked and various scenarios in Encryption campaign are unlocked without playing any scenarios in that campaign
